ass_man/views/DecommissionedViewSet.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import viewsets
+from ass_man.models import Decommissioned
+from ass_man.serializers.decommissioned_serializers import DecommissionedSerializer
+
+class DecommissionedViewSet(viewsets.ModelViewSet):
+    queryset = Decommissioned.objects.all()
+    serializer_class = DecommissionedSerializer
```

Comment:
we should look at only allowing certain actions here (list and retrieve)/(GET only) -- look at how we determine permission classes for some of the other viewsets

Reply:
bump